litellm/llms/vertex_ai/vertex_llm_base.py

```diff
@@ -301,7 +301,7 @@ def _check_custom_proxy(
         """
         if api_base:
             if custom_llm_provider == "gemini":
-                url = "{}:{}".format(api_base, endpoint)
+                url = "{}/{}".format(api_base.rstrip("/"), endpoint.lstrip("/"))
```

Comment:
`api_base` + `endpoint` seems to still be missing the model information

If `api_base=https://api.example.com/v1beta`, appending the `generateContent` endpoint results in the URL `https://api.example.com/v1beta:generateContent`.